internal/storage/gcs/bucket.go

```diff
@@ -26,8 +26,10 @@ type BucketType int
 
 // BucketType enum values.
 const (
-	NonHierarchical BucketType = iota
+	Nil BucketType = iota
```

Comment:
please add a comment here Default value Nil represents that bucket type is not set